### PR TITLE
MAKE-1111: Trigger signal processing job in cedar when YCSB added manually

### PR DESCRIPTION
**Pull request by ancostas**

### CODE REVIEW

File: rpc/internal/perf_service.go [hunk 1]
```diff
@@ -106,6 +106,19 @@ func (srv *perfService) AttachRollups(ctx context.Context, rollupData *RollupDat
...
 
+	id := model.PerformanceResultSeriesId{
+		Project: record.Info.Project,
+		Variant: record.Info.Variant,
+		Task:    record.Info.TaskName,
+		Test:    record.Info.TestName,
+	}
+
+	processingJob := units.NewRecalculateChangePointsJob(id)
+	err := srv.env.GetRemoteQueue().Put(ctx, processingJob)
```
Comment (tychoish) [inline rpc/internal/perf_service.go]: combine 116 and 117

File: rpc/internal/perf_service.go [hunk 2]
```diff
@@ -106,6 +106,19 @@ func (srv *perfService) AttachRollups(ctx context.Context, rollupData *RollupDat
 		return nil, newRPCError(codes.InvalidArgument, errors.Wrapf(err, "problem attaching rollup data for perf result '%s'", record.ID))
 	}
 
+	id := model.PerformanceResultSeriesId{
```
Comment (tychoish) [inline rpc/internal/perf_service.go]: two things: 

1. A nit. I'd prefer to have the d in ID capitalized. 
2. is there a way to have a method to produce this kind of object from the record/info type? I've seen this pattern used a lot, and it seems easy to get it wrong....

Comment (ancostas) [reply rpc/internal/perf_service.go]: Capitalized ID and added a `ToPerformanceResultSeriesId` to `PerformanceResultInfo`.

File: rpc/internal/perf_service.go (hunk 1 above)
Comment (ancostas) [reply rpc/internal/perf_service.go]: Go fmt file is pushing them back apart for me (line length?). Not for you?

**Pull request merged**